matplotlibrc

```diff
@@ -0,0 +1 @@
+backend: Agg
```

Comment:
I think this file should be created automatically by `make`, and ignored by `git`.

Since all of the backends now are set to `Agg`, we may also consider removing that logic from the `Makefile`, and replace it with this file

Reply:
Ok, leave it as it is now, but maybe we change it later